presentation/components/package.json

```diff
@@ -123,7 +120,7 @@
     "extends": "./node_modules/@itwin/build-tools/.nycrc",
     "check-coverage": true,
     "statements": 100,
-    "functions": 100,
+    "functions": 99,
```

Comment:
why did this have to be reduced?

Reply:
Exporting `PRESENTATION_TREE_NODE_KEY` in the barrel broke it, spoke with @saskliutas who said he'll fix this after this pr goes through.

https://github.com/istanbuljs/nyc/issues/1286